src/templates/rule.js

```diff
@@ -148,19 +68,47 @@ export default ({ data }) => {
 						}}
 					/>
 					{/* glossary */}
-					{renderGlossaryUsed(slug)}
+					{getGlossaryUsed(slug, allGlossary)}
+					{/* changelog */}
+					{getChangelog(ruleChangelog, repository.url)}
+					{/* acknowledgements */}
```

Comment:
- ensure authors are with in a list, lose the `@`, and check the data for contributors in package.json is right.